project/UI/CLIKit/CommandLineTool.swift

```diff
@@ -0,0 +1,13 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+import ArgumentParser
+import NefCarbon
+import Bow
+import BowEffects
+
+public struct CommandLineTool<T: ParsableCommand> {
+    public static func main() {
+        _ = CarbonApplication { T.main() }
+    }
+}
```

Comment:
You are embedding every command into a CarbonApplication, no only the Carbon-related ones. Is this the intended behavior?

Reply:
yep, it is!
In the end, Command-Line will run wrapped in `NSApplication` context, and we get AppKit powerful.